atd-vzv/src/constants/time.js

```diff
@@ -15,27 +25,12 @@ export const dataEndDate = moment()
   .subtract(1, "month")
   .endOf("month");
 
-// Common time variables
-export const today = moment().format("YYYY-MM-DD");
-export const thisYear = moment().format("YYYY");
-export const oneYearAgo = moment()
-  .subtract(1, "year")
+// Summary time data
+export const summaryCurrentYearStartDate = dataEndDate
+  .clone() // Moment objects are mutable
```

Comment:
good to know about moment mutability!

Reply:
Yep, I felt kind of 🙃when I was trying to do math with a `moment` and kept mutating it and messing up existing references to it. Found [this](https://momentjs.com/guides/#/lib-concepts/mutability/) guide which I have never actually run into since it is separate from the docs.